storage/rocksdb/rdb_sst_info.cc

```diff
@@ -392,13 +393,23 @@ int Rdb_sst_info::open_new_sst_file() {
 }
 
 void Rdb_sst_info::commit_sst_file(Rdb_sst_file_ordered *sst_file) {
+  m_commiting_threads_mutex.lock();
```

Comment:
consider std::lock_guard<std::mutex>? we use `RDB_MUTEX_LOCK_CHECK` macros elsewhere in this file

Reply:
>     1. In `Rdb_sst_file_ordered::commit()`, if `m_use_stack` is true, it will write kv from stack.
> 
>     2. In `Rdb_sst_file_ordered::Rdb_sst_file::commit()`, it calls `m_sst_file_writer->Finish()`, in which will write file and call fsync.
> 
> 
> In our private branch, when `m_use_stack` is true(reverse cf), `create index` time reduces 30+%.

Rdb_index_merge and Rdb_sst_file_ordered both use cf's comparator, when Rdb_index_merge write to Rdb_sst_file_ordered, the keys should be in cf's increasing order, m_use_stack should be false in this case.